src/api/comments.ts

```diff
@@ -0,0 +1,32 @@
+import { BASE_URL, request } from './api';
+import { Comment } from '../types/Comment';
+
+export const getPostComments = async (postId: number): Promise<Comment[]> => {
+  const comments = await request(`${BASE_URL}/comments`);
+
+  return comments.filter((comment: Comment) => comment.postId === postId);
```

Comment:
use API ability to get needed comments
<img width="806" alt="image" src="https://user-images.githubusercontent.com/64437769/158980884-72ecbca1-d937-46a1-943a-5f528207eb95.png">

Reply:
done